<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookImformation</class>
 <widget class="QWidget" name="bookImformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>bookImformation</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>641</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">backgorund-color:#f7f7f5;
font-weight:bold;
letter-spacing:1;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#717170;
font-weight:blod;</string>
    </property>
    <property name="text">
     <string>书籍信息</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>601</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">opacity:0.7;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#717170;
font-weight:blod;</string>
    </property>
    <property name="text">
     <string>借阅信息</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>601</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">opacity:0.7;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_001">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#717170;
font-weight:blod;</string>
    </property>
    <property name="text">
     <string>书名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_002">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#717170;
font-weight:blod;</string>
    </property>
    <property name="text">
     <string>ISBN：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_003">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#717170;
font-weight:blod;</string>
    </property>
    <property name="text">
     <string>作者：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_004">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#717170;
font-weight:blod;</string>
    </property>
    <property name="text">
     <string>出版社：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_100">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#717170;
font-weight:blod;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_200">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#717170;
font-weight:blod;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_300">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#717170;
font-weight:blod;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_400">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#717170;
font-weight:blod;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>611</width>
      <height>141</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>书号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>借阅时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>归还时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>状态</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#717170;
font-weight:blod;</string>
    </property>
    <property name="text">
     <string>书籍详细信息</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>611</width>
      <height>121</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>书号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>状态</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>所在位置</string>
     </property>
    </column>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>601</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">opacity:0.7;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
